packages/admin-ui-components/src/loading/loading.tsx

```diff
@@ -99,7 +99,7 @@ export const Loading: React.FC<LoadingProps> = props => {
 
     return (
       <div className={cx("alerts-container", props.errorClassName)}>
-        {errorAlerts}
+        {React.Children.toArray(errorAlerts)}
```

Comment:
why is this necessary?

Reply:
It fixes React error about missing `key` attribute for array of elements.
`React.Children.toArray` accepts an array of react elements and allows to set automatically `key` attribute if it's missed.